### DISCUSSION TITLE
"Start chat" button should open the chat pannel

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19185] [Q&A] [answered]
Current behaviour just adds player to a chat list, but I think the button should open the chat pannel first.

https://user-images.githubusercontent.com/86934170/179395612-03c7734d-c0b4-4a8b-8473-ceee2ea0b5bf.mp4

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/19202
